src/utils/extensions/telemetry/telemetry.ts

```diff
@@ -0,0 +1,106 @@
+import { kubevirtConsole as kvConsole } from '@kubevirt-utils/utils/utils';
```

Comment:
why changing its name?

Reply:
I don't remember why I did that. I changed it back.